src/sentry/integrations/slack/views/__init__.py

```diff
@@ -0,0 +1,17 @@
+from django.urls import reverse
+from django.views.decorators.cache import never_cache as django_never_cache
+
+from sentry.utils.http import absolute_uri
+from sentry.utils.signing import sign
+from sentry.utils.types import Any
+from sentry.web.decorators import EndpointFunc
+
+
+def never_cache(view_func: EndpointFunc) -> EndpointFunc:
+    """TODO(mgaeta): Remove this HACK once Django has a typed version."""
+    return django_never_cache(view_func)  # type: ignore
+
+
+def build_linking_url(endpoint: str, **kwargs: Any) -> str:
+    """TODO(mgaeta): Remove this HACK once sentry/utils/http.py is typed."""
+    return absolute_uri(reverse(endpoint, kwargs={"signed_params": sign(**kwargs)}))  # type: ignore
```

Comment:
I'm not sure I understand the need for this. If `absolute_uri` returns a string, you could cast this call to a string in your code.

Reply:
I've updated the function cast the value while saving to a variable, but it still feels like a hack.

Reply:
Sorry, what I meant was if all this function does is cast the value I'm not sure I see the point of the function at all. Move the cast into the main code body.

Reply:
Lol I understand now. I think I'll keep the function because I like that its encapsulates creating the URL and is used 3 times so far.